test/model_traits.jl

```diff
@@ -72,5 +72,9 @@ struct FooMeasure <: MLJType end
     @test name(Float64) == "Float64"
 
     df = DataFrame(a=randn(2), b=randn(2))
-    @test string(M.coretype(typeof(df))) == "DataFrame"
+    if VERSION < v"1.6-"
+        @test string(M.coretype(typeof(df))) == "DataFrame"
+    else
+        @test string(M.coretype(typeof(df))) == "typename(DataFrame)"
+    end
```

Comment:
Don't we want the output to be the same, independent of julia version? Namely, that "DataFrame" is returned?

Reply:
Just FYI, we *do* want this to work for types other than subtypes of `MLJType`, for instance, measures defined in `LossFunctions.jl`, which are imported by MLJBase.

Reply:
The fix is applied to `name` so would apply in all cases and all versions, just in some case the replace would not have an effect. I left this test more as an indicator of why the replace trick was necessary?

Reply:
I added a line to the test to make this more explicit

Reply:
Okay, I see. I thought maybe `coretype` (whose behaviour is different under 1.6 in this PR and before) was used somewhere outside MMI, but from a grep it appears I was mistaken.

Many thanks.